source/_integrations/fireservicerota.markdown

```diff
@@ -0,0 +1,226 @@
+
+You will need a FireServiceRota or BrandweerRooster account.
+
+<div class='note'>
+
+A word of caution: Do not solely rely on this integration for your emergency calls!
+
+</div>
+
+This integration provides the following platforms:
+
+- Sensor: Incoming emergency calls. Metadata contains _among other data_ the location of the incident and a text-to-speech URL. The integration uses a WebSocket client connection with the service to ensure a minimum delay.
+- Binary Sensor: Your current duty status (as scheduled via the FireServiceRota mobile app and/or website).
+- Switch: Enabled for 30 minutes after an emergency call. ‘on’ represents a confirmed response. Use this to automate your emergency call response and save valuable seconds.
+
+On how to write automations using these platform read the 'Advanced Configuration' section below.
+
+## Configuration
+
+1. From Home Assistant, navigate to ‘Configuration’ then ‘Integrations’. Click the plus icon and type/select ‘FireServiceRota’.
+1. Choose your platform `BrandweerRooster` or `FireServiceRota`.
+1. Enter your login credentials.
+
+1. Click the Save button.
+
+## Entities
+
+The following entity types are created:
+
+### Incidents Sensor
+
+| Value | Description |
+| ----- | ----------- |
+| `value` | Contains the complete incident text message.|
+
+| Attribute | Description |
+| --------- | ----------- |
+| `trigger` | Type of trigger, `new` or `update`.|
+| `state` | The state of the incident. |
+| `created_at` | Date and time when incident was created.|
+| `message_to_speech_url` | The URL of the mp3 file containing the spoken text of the incident.|
+| `prio` | Priority of the incident, `a1`, `a2`, `b1` or `b2`.|
+| `type` | Type of incident, e.g. `incident_alert`.|
+| `responder_mode` | Modes of response, e.g. `available_in_schedule_is_acknowledgment`.|
+| `can_respond_until` | Date and time until response is accepted.|
+| `latitude` | The Latitude of the incident.|
+| `longitude` | The Longitude of the incident.|
+| `address_type` | Type of address, e.g. `home`.|
+| `formatted_address` | Address in string format.|
+
+### Duty Binary Sensor
+
+| Value | Description |
+| ----- | ----------- |
+| `value` | `on` = on duty. `off` = no duty scheduled.|
+
+| Attribute | Description |
+| --------- | ----------- |
+| `start_time` | Start date and time of duty schedule.|
+| `end_time` | End date and time of duty schedule.|
+| `available` | `true` or `false`.|
+| `active` | `true` or `false`.|
+| `assigned_function_ids` | Function id's, e.g. `540`.|
+| `skill_ids` | Skill id's, e.g. `6, 8`.|
+| `type` | Type, e.g. `standby_duty`.|
+| `assigned function` | Assigned function, e.g. `Chauffeur`.|
+
+### Incident Response Switch
+
+| Value | Description |
+| ----- | ----------- |
+| `value` | `on` = response is acknowledged. `off` = the response is rejected.|
```

Comment:
As mentioned before, I really think we do not need a table here.
It makes more sense to have a written introduction about the sensor and mention the values in that description. (Removing the whole table).

Reply:
@frenck I change them, what do you think?